### PR TITLE
JS3week2 without visual promises

### PR DESCRIPTION
**Pull request by OlenaKasian**

### CODE REVIEW

File: javascript/javascript3/week2/fetch-movie.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const movies = fetch(`https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json`)
+.then((res) => res.json())
+.then ((data) => {
+    const badMovies = data.filter(movies => movies.rating < 5);
```
Comment (qamarfj) [inline javascript/javascript3/week2/fetch-movie.js:4]: I would use variables names  here like this.
  .then ((movies) => {
    const badMovies = movies.filter(movie => movie.rating < 5);

File: javascript/javascript3/week2/exercises2-3.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+//Rewrite setTimeout
+
+function mySetTimeOut() {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve()
+    }, 1000);
+  })
+}
+mySetTimeOut(3).then(() => {
```
Comment (qamarfj) [inline javascript/javascript3/week2/exercises2-3.js:28]: Here you are passing one argument but in function definition you are not using any parameter.

File: javascript/javascript3/week2/promise-visual/main.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    )
+    console.log('third element')
+}
+
+function moveAllAtOnce() {
+    Promise.all([
+        moveElement(marks[0], { x: 20, y: 300 }),
+        moveElement(marks[1], { x: 400, y: 300 }),
+        moveElement(marks[2], { x: 400, y: 20 }),
+    ])
```
Comment (qamarfj) [inline javascript/javascript3/week2/promise-visual/main.js:27]: Here you missing .then block.

**Review by qamarfj — COMMENTED** [review]
Hi @OlenaKasian , you have done great work! .
I left some comments in code which are very small things but need to consider.
In Visual promise assignment the ellements are not reaching on target positions. Please Note that because the elements start with a random position, it is up to you to calculate how much each element needs to be moved in x/y directions

File: javascript/javascript3/week2/fetch-movie.js (hunk 1 above)
Comment (OlenaKasian) [reply javascript/javascript3/week2/fetch-movie.js:4]: Thank you, it looks better

File: javascript/javascript3/week2/promise-visual/main.js (hunk 3 above)
Comment (OlenaKasian) [reply javascript/javascript3/week2/promise-visual/main.js:27]: I will add!

**Comment by OlenaKasian** [comment]
Thank you @qamarfj for the code review!

**Pull request merged**